openquake/calculators/tests/gmf_ebrisk_test.py

```diff
@@ -79,6 +80,26 @@ def test_ebr_2(self):
         totloss = alt['loss'].sum()
         aae(totloss, 20210.27, decimal=2)
 
+    @attr('qa', 'risk', 'gmf_ebrisk')
+    def test_case_4(self):
+        # a simple test with 1 asset and two source models
+        self.run_calc(case_4.__file__, 'job_haz.ini')
+        calc0 = self.calc.datastore  # event_based
+        self.run_calc(case_4.__file__, 'job_risk.ini',
+                      calculation_mode='event_based_risk',
+                      hazard_calculation_id=str(calc0.calc_id))
+        calc1 = self.calc.datastore  # event_based_risk
+        self.run_calc(case_4.__file__, 'job_risk.ini',
+                      hazard_calculation_id=str(calc0.calc_id))
+        calc2 = self.calc.datastore  # event_based_risk
+        [fname] = export(('agg_loss_table', 'csv'), calc1)
+        self.assertEqualFiles('expected/' + strip_calc_id(fname), fname,
+                              delta=1E-5)
+        #[fname] = export(('agg_loss_table', 'csv'), calc2)
```

Comment:
Can this be removed?

Reply:
This is the part that it is breaking the test and that I will fix in the future PR. The whole purpose of this PR is to expose this issue ;-)